packages/workbox-build/src/lib/populate-sw-template.js

```diff
@@ -0,0 +1,56 @@
+const template = require('lodash.template');
+
+const errors = require('./errors');
+const runtimeCachingConverter = require('./runtime-caching-converter');
+
+module.exports = ({
+  cacheId,
+  clientsClaim,
+  directoryIndex,
+  handleFetch,
+  ignoreUrlParametersMatching,
+  importScripts,
+  manifestEntries,
+  navigateFallback,
+  navigateFallbackWhitelist,
+  runtimeCaching,
+  skipWaiting,
+  swTemplate,
```

Comment:
Why is this being passed in?

Couldn't this be a static string template in workbox-build that would be passed to lodash?

Reply:
My initial motivation for taking this in as a string was to avoid having to make a `fse.readFile()` call to read it in to memory, since that wouldn't work nicely with Webpack. And including

```js
const SW_TEMPLATE = `really long template string`;
```

inline within another module seems kind of messy.

After thinking about it some more, what I could do is change things so that the `swTemplate` is standalone in its own module file, that consists of

```js
module.exports = `really long template string`;
```

I think that would play nicely with Webpack and still let it be a standalone thing, and theoretically easy to test via `proxyquire()`. I'll make that change.